src/popover/utils/positions.ts

```diff
@@ -265,11 +267,17 @@ export function calculatePosition({
       ? getIntersection([rect, viewport])
       : getIntersection([rect, viewport, container]);
 
-    const fitsWithoutOverflow =
-      visibleArea && visibleArea.inlineSize === body.inlineSize && visibleArea.blockSize === body.blockSize;
+    // When min height is set, the popover is considered fitting the container if the available space
+    // is the same or larger than min height, even if it means the scrollbar is needed.
```

Comment:
Why? This looks to me like `minHeight` is used as a max rather than a min.

Reply:
The idea was to force the popover into its position when its height is at least minHeight. For example, if the popover content is ~500px, the minHeight is set to 200px, and the available space is 300px - the popover will be rendered to the preferred position because its actual height is more or equal to min height. If the available space is less than minHeight - the popover will be rendered to the next suitable position instead.

Reply:
I still don't understand. In this example, the actual popover content height (500) is more than the minHeight (200) so it should not be affected/constrained by it, right (minHeight is a min, not a max)? So how should the minHeight of 200 here affect the positioning, if the actual popover height will be the same regardless of minHeight?

Reply:
Is `minHeight` something like "minimum vertical space that needs to be available in order let the positioning logic override the preferred position"?

Reply:
Yes, exactly. Would you recommend a better name for the prop?

Reply:
Yes :)

Reply:
Maybe `minVisibleBlockSize` or `minVisibleHeight` is marginally better? The whole positioning logic concept is not easy to convey in the name but at least the "visible" part hopefully gives some hint to it.

My main problem with just `minHeight` is, it would suggest that it is for setting the actual popover height to a minimum value, e.g, adding white space below the inner content if necessary.

Reply:
If you change it, please make sure to change it also wherever it is referenced elsewhere like comments and test descriptions.

Reply:
Thanks, I will change it to minVisibleBlockSize 👍